docs/overrides/main.html

```diff
@@ -26,7 +26,7 @@
   
         async function main() {
           globalThis.pyodide = await loadPyodide({
-            indexURL: "https://cdn.jsdelivr.net/pyodide/v0.18.1/full/",
+            indexURL: "https://cdn.jsdelivr.net/pyodide/v0.19.1/full/",
```

Comment:
This is unrelated to this update (this version of pyodide will still run Python 3.9), but it will make it easier to update to a pyodide version that uses 3.10 [once it's released](https://github.com/pyodide/pyodide/issues/2190).